richardcoffey/minecraft-website
# Reformatting

This PR introduces re-formatting, proper indention, and some css improvements.

Steps to Test:
* Render each .html file.
* Ensure that the .html file rendered state doesn't change. Meaning, the style and appearance of the page shouldn't change with this update.
* Read through html and css files, and ensure that formatting is correct.
